### PR TITLE
[bitnami/janusgraph] Update version of Cassandra's chart

### PR DESCRIPTION
**Pull request by dgomezleon**
### Description of the change

Update Cassandra's chart to version 12

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/janusgraph/Chart.yaml [hunk 1]
```diff
@@ -5,19 +5,21 @@ annotations:
...
     - name: jmx-exporter
       image: docker.io/bitnami/jmx-exporter:1.0.1-debian-12-r6
     - name: os-shell
-      image: docker.io/bitnami/os-shell:12-debian-12-r28
+      image: docker.io/bitnami/os-shell:12-debian-12-r30
 apiVersion: v2
 appVersion: 1.0.0
 dependencies:
-- condition: storageBackend.cassandra.enabled
+- condition: cassandra.enabled
```
Comment (juan131) [inline bitnami/janusgraph/Chart.yaml:19]: Revert this change, the parameter `storageBackend.cassandra.enabled` is still the one to use.

Comment (dgomezleon) [reply bitnami/janusgraph/Chart.yaml:19]: Good catch

**Review by juan131 — APPROVED** [review]
LGTM

**Review by juan131 — APPROVED** [review]
